src/repl/scala/tools/nsc/interpreter/MemberHandlers.scala

```diff
@@ -162,36 +170,34 @@ trait MemberHandlers {
   }
 
   class TermMacroHandler(member: DefDef) extends MacroHandler(member) {
-    def notification(req: Request) = s"defined term macro $name: ${req.typeOf(name)}"
+    def notification(req: Request) = s"// defined term macro ${req.defTypeOf(name)}"
   }
 
   class AssignHandler(member: Assign) extends MemberHandler(member) {
     override def resultExtractionCode(req: Request) =
-      codegenln(s"mutated ${member.lhs}")
+      codegenln(s"// mutated ${member.lhs}")
   }
 
   class ModuleHandler(module: ModuleDef) extends MemberDefHandler(module) {
     override def definesTerm = Some(name.toTermName)
     override def definesValue = true
 
-    override def resultExtractionCode(req: Request) = codegenln("defined object ", name)
+    override def resultExtractionCode(req: Request) = codegenln("// defined object ", name)
   }
 
   class ClassHandler(member: ClassDef) extends MemberDefHandler(member) {
     override def definedSymbols = List(symbol, symbol.companionSymbol) filterNot (_ == NoSymbol)
     override def definesType = Some(name.toTypeName)
     override def definesTerm = Some(name.toTermName) filter (_ => mods.isCase)
 
-    override def resultExtractionCode(req: Request) =
-      codegenln("defined %s %s".format(keyword, name))
+    override def resultExtractionCode(req: Request) = codegenln(s"// defined $keyword $name")
   }
 
   class TypeAliasHandler(member: TypeDef) extends MemberDefHandler(member) {
     private def isAlias = mods.isPublic && treeInfo.isAliasTypeDef(member)
     override def definesType = Some(name.toTypeName) filter (_ => isAlias)
 
-    override def resultExtractionCode(req: Request) =
-      codegenln("defined type alias ", name) + "\n"
+    override def resultExtractionCode(req: Request) = codegenln(s"// defined type alias $name")
```

Comment:
Some of these are `// defined` and some look like their declaration (vals, vars, defs). Can we use one style for all?

Reply:
Also, this message is inaccurate for abstract types.

Reply:
so would you just have `type Foo`?

Reply:
That, or make abstract types say `abstract type Foo`